docs/plugins/pytorch_dali_proxy.rst

```diff
@@ -0,0 +1,164 @@
+PyTorch DALI Proxy
+==================
+
+Overview
```

Comment:
Perhaps even in the overview, I'd like to spare the users some disillusionment and state explicitly that DALI cannot be followed by any further processing steps.